.jenkins/common_job_properties.groovy

```diff
@@ -155,6 +169,12 @@ class common_job_properties {
     setPullRequestBuildTrigger(context, comment)
   }
 
+  static def setPreCommitWithSuccessComment(def context,
```

Comment:
This doesn't seem necessary -- you have just one default and it is the last parameter.